bamboo/create-release.sh

```diff
@@ -5,26 +5,19 @@ source .bamboo_env_vars || true
 . ./bamboo/set-bamboo-env-variables.sh
 . ./bamboo/abort-if-not-pr.sh
 
+# Build TF module artifacts
+. ./bamboo/create-release-artifacts.sh
```

Comment:
@Jkovarik This should work, right? Not sure how to test this before actually doing a release

Reply:
We can test the bash is good by running it in the container bamboo would be running it in, if we're worried about cross mount perms (this *probably isn't an issue) we could mock something up in bamboo, but that's probably not needed.

Reply:
You need to make sure the permissions on the referenced script have u+x in the repo though.